### DISCUSSION TITLE
WebViewControl-Avalonia pipe line build failed

### DISCUSSION THREAD

**Discussion by Nikhilthambi** [AvaloniaUI/Avalonia#10674] [General]
I am using webview controller ("WebViewControl-Avalonia" Version="2.106.6") for rendering html in avalonia (windows)

<PackageReference Include="WebViewControl-Avalonia" Version="2.106.6" />

[WebViewControl-Avalonia](https://www.nuget.org/packages/WebViewControl-Avalonia)

pipeline build command:-

dotnet publish --sc -r win-x64 -o "AvaloniaTool$(VERSION)_x64" -c Release -p:DebugType=None -p:DebugSymbols=false

I know some problems are going on regarding WebViewControl-Avalonia but this is something different.
In rider debug mode and release mode build is perfect but while build in pipeline build is failing due to Xilium.CefGlue.BrowserProcess.exe

`Application: AvaloniaTool.exe
CoreCLR Version: 6.0.21.52210
.NET Version: 6.0.0
Description: The process was terminated due to an unhandled exception.
Exception Info: System.TypeInitializationException: The type initializer for 'Xilium.CefGlue.CefRuntime' threw an exception.`

`when use WebViewControl-Avalonia Version="2.91.29" build is success but webview controller is not working and couldn't render html`

I tried some downgrade versions of WebViewControl-Avalonia but still the problem is exist

and install following supporting libraries for Xilium.CefGlue 

`"CefGlue.Avalonia" Version="106.5249.6",
"VitalElement.CefGlue.Core" Version="0.9.0-build20-alpha",
"VitalElement.CefGlue.Core.Avalonia"`

Could you please suggest some fixes and I am curios about if some one already faced or fixed something similar like this issue, all suggestion and helping would be appreciate

Thanks

**Comment by maxkatz6** [comment]
We don't maintain this control and can't help you much.
Try to ask for help here https://github.com/OutSystems/WebView
